samplescalaproject/src/main/scala/example/Hello.scala

```diff
@@ -0,0 +1,9 @@
+package example
+
+object Hello extends Greeting with App {
+  println("Welcome to 30 days of Scala")
+}
+
+trait Greeting {
+  lazy val greeting: String = "hello"
```

Comment:
the changes are supposed to be made on the greeting variable 
```suggestion
  lazy val greeting: String = "Welcome to 30 days of Scala"
```